src/api.raml

```diff
@@ -185,6 +185,37 @@ traits:
               required: false
               example: [article/1234, interview/5678]
 
+/annotations/{id}:
+    type: collection
+    description: |
+        Annotations
+    uriParameters:
+        id:
```

Comment:
That looks like the ID of an annotation. Instead, maybe we should have `by` as a query string parameter, and make it required for now (so if we ever allow lists by everyone, make it optional). (Or even by multiple authors, allow it to be multiple.)

Reply:
Will make this adjustment.